src/Serilog.Ui.Web/assets/index.html

```diff
@@ -212,7 +213,7 @@ <h5 class="modal-title" id="exampleModalLabel">Change Page</h5>
             window.config = configObject;
         } catch (e) {
             console.warn('SerilogUI Config not received correctly! Using defaults');
-            window.config = JSON.parse('{"routePrefix":"serilog-ui","authType":"Jwt"}');
+            window.config = JSON.parse('{"routePrefix":"serilog-ui","authType":"Jwt"');
         }
     </script>
 </body>
```

Comment:
```suggestion
         <meta name="dummy" content="%(BodyContent)">
</body>
```
if you like the BodyContent suggestion, I'd add the option here